executor/ddl_test.go

```diff
@@ -183,6 +183,15 @@ func (s *testSuite) TestAddNotNullColumnNoDefault(c *C) {
 	tk.MustExec("set sql_mode=''")
 	tk.MustExec("insert nn (c1) values (3)")
 	tk.MustQuery("select * from nn").Check(testkit.Rows("1 0", "2 0", "3 0"))
+
+	tk.MustExec("alter table nn add column (c3 int default -1, c4 int)")
```

Comment:
1. Error cases in ddl_worker_test.TestColumnError.
2. More position column cases in column_change_test.testAddMultipleColumns.
3. Type cases in ddl_test.TestAlterTableAddMultipleColumns.